common/config/azure-pipelines/templates/setup-integration-users.yaml

```diff
@@ -18,6 +18,8 @@ steps:
     echo "##vso[task.setvariable variable=test_bing_maps_key;]$bing_key"
     echo "##vso[task.setvariable variable=test_mapbox_key;]$mapbox_key"
     echo "##vso[task.setvariable variable=imjs_cesium_ion_key;]$cesium_ion_key"
+    echo "##vso[task.setvariable variable=mes_testuser;]$mes_user"
```

Comment:
do we need a separate user account & pw?
couldnt use the exisisting test users?

Reply:
I'm currently using this separate user since it was one I know we had control over and could therefore give access to the right imodels to use in the test, etc.

Reply:
I think we should work on getting the test accounts access to these models, instead of creating another set of credentials

Reply:
The existing IMJS_REGULAR_TEST_USER now has access to the model we will use to test this.